plugins/newsdownloader.koplugin/main.lua

```diff
@@ -844,7 +792,7 @@ function NewsDownloader:editFeedAttribute(id, key, value)
 end
 
 function NewsDownloader:updateFeedConfig(id, key, value)
-    local kv = self.kv
+    local UI = require("ui/trapper")
     -- Because this method is called at the menu,
```

Comment:
Is there a particular reason you're using Trapper for some simple info messages? I'll merge this PR now but that looks a bit suspicious.

Reply:
It could just be my own lack of total understanding of trapper and info messages. I think I'm missing a sense of the best practices for these UI boxes.

Reply:
As described in the manual (and generally all throughout the code…? ;-) )
http://koreader.rocks/doc/modules/ui.widget.infomessage.html
http://koreader.rocks/doc/modules/ui.trapper.html

Reply:
tl;dr Don't use Trapper unless you need to.

Reply:
I wondered too when I saw the previous PR - but Trapper was already used by NewsDownloader, so it's possible @roygbyte did not introduce another use of it and it was just some refactoring of the already there usage :)
Trapper is good (and needed) when you have some multiple steps long processing happening, and you want to give feedback and ask questions all along.

Reply:
Yes, NewsDownloader does exactly that kind of thing when it's downloading everything. But for a method called `updateFeedConfig()` it's presumably unnecessary. ^_^

Reply:
Unnecessary, but wonderfully lavish, no? :P